dev-docs/modules/audigentRtdProvider.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: page_v2
+title: Audigent Segmentation RTD Module
+description: Audigent Segmentation RTD Module
+display_name: Audigent Segmentation
+module_code : audigent
+module_type : rtd
+display_name : Audigent
+enable_download : true
+sidebarType : 1
+---
+
+# Audigent Segmentation RTD Module
+{:.no_toc}
+
+* TOC
+{:toc}
+
+## Overview
+
+Audigent is a next-generation data management platform and a first-of-a-kind 
+"data agency" containing some of the most exclusive content-consuming audiences 
+across desktop, mobile and social platforms.
+
+This real-time data module provides quality user segmentation that can be 
+attached to bid request objects destined for different SSPs in order to optimize 
+targeting. Audigent maintains a large database of first-party Tradedesk Unified 
+ID, Audigent Halo ID and other id provider mappings to various third-party 
+segment types that are utilizable across different SSPs.  With this module,
+these segments can be retrieved and supplied to the SSP in real-time during
+the bid request cycle.
+
+### Usage
```

Comment:
Who is this usage for? I'm unclear what parts here are for a publisher and which parts are for bid adapters that want to utilize your data.

Reply:
It's for the publishers. https://github.com/prebid/Prebid.js/pull/5903/commits/87d7cc40b496b0cd1e5e5376aba163177fefb33a adds segments to the first party data segment object for publishers to draw from as they update their adapters to follow that standard.  However, until they do, we are opting to provide RTD into the bid stream by augmenting bids to provide API compatible segments on bid objects per-adapter prior to their being sent to respective bidders.